### PR TITLE
docs(issues): update issues list + add installer issue

### PR DESCRIPTION
**Pull request by Valastiri**
### Location

- docs/fbw-a32nx/support/reported-issues.md

Discord username (if different from GitHub): Valastiri#8902

### CODE REVIEW

**Comment by Valastiri** [comment]
pending #support-ops consideration + possible additions to PR

File: docs/fbw-a32nx/support/reported-issues.md [hunk 1]
```diff
@@ -570,6 +550,26 @@ TEMPLATE
 
 ---
 
+??? tip "Ailerons When Using the Keyboard"
```
Comment (frankkopp) [inline docs/fbw-a32nx/support/reported-issues.md]: I would move this completely to the Fixed section as it is no longer an issue for users.

File: docs/fbw-a32nx/support/reported-issues.md [hunk 2]
```diff
@@ -655,6 +655,30 @@ TEMPLATE
 
 ---
 
+## FBW Installer Issue
```
Comment (frankkopp) [inline docs/fbw-a32nx/support/reported-issues.md]:
```suggestion
## FBW Installer Issues
```

File: docs/fbw-a32nx/support/reported-issues.md [hunk 3]
```diff
@@ -655,6 +655,30 @@ TEMPLATE
...
+
+    ![sample installer image](https://media.discordapp.net/attachments/926586416820011098/929800696302276718/unknown.png)
+
+    ^^Root Cause^^
+
+    Small issue with our installer and the A32NX files
+
+    ^^Possible Solution or Workaround^^
+
+    Please delete the flybywire folder in your community folder and redownload the aircraft.
```
Comment (frankkopp) [inline docs/fbw-a32nx/support/reported-issues.md:658]:
```suggestion
    Please delete the flybywire folder in your community folder and redownload the aircraft.
    
    This should only be necessary once. 
```

**Review by frankkopp — CHANGES_REQUESTED** [review]

**Review by frankkopp — APPROVED** [review]

**Pull request merged**